spyderlib/cli_options.py

```diff
@@ -39,5 +40,8 @@ def get_options():
     parser.add_option('--profile', action='store_true', default=False,
                       help="Profile mode (internal test, "
                            "not related with Python profiling)")
+    parser.add_option('--travis', dest="travis", action='store_true',
+                      default=False,
+                      help="Run a Qtimer to stop the GUI app in travis-ci")
```

Comment:
By using an environment variable. Take a look at how I define and use the `DEV` variable in `bootstrap.py` and then in `spyder.py`